src/Sarif.Viewer.VisualStudio/ErrorList/ColumnFilterer.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Shell.TableControl;
+
+namespace Microsoft.Sarif.Viewer.ErrorList
+{
+    // This class allows the caller to filter specified values from specified columns.
+    // It keeps track of which column/value pairs have been filtered, so that it only
+    // does so once per column/value pair. The idea is that if the user ever checks the
+    // box in the filter UI to show a certain value in a certain column, we should never
+    // hide that column again.
```

Comment:
>again [](start = 24, length = 5)

@mikefan@microsoft.com Note that this comment contains the information you asked for above the definitions of `ErrorListService.ShowFilteredSuppressionsStateColumn` and `ErrorListService.ShowFilteredCategoryColumn`. I will repeat it in those locations.